nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -1548,24 +1546,55 @@ def __parse_item_rttm(self, line: str, manifest_file: str) -> Dict[str, Any]:
             raise ValueError(
                 f"Manifest file has invalid json line " f"structure: {line} without proper audio file key."
             )
+
+        # Audio file handling
         if isinstance(item['audio_file'], list):
-            item['audio_file'] = [os.path.expanduser(audio_file_path) for audio_file_path in item['audio_file']]
+            audio_file_list = []
+            for single_audio_file in item['audio_file']:
+                if not os.path.exists(single_audio_file):
+                    raise FileNotFoundError(f"Audio file not found: {single_audio_file}")
+                else:
+                    audio_file_list.append(get_full_path(audio_file=ingle_audio_file, manifest_file=manifest_file))
+            item['audio_file'] = audio_file_list
+        else:
+            if not os.path.exists(item['audio_file']):
+                raise FileNotFoundError(f"Audio file not found: {item['audio_file']}")
+            else:
+                item['audio_file'] = get_full_path(audio_file=item['audio_file'], manifest_file=manifest_file)
+
+        # If the name `rttm_file` is not present in the manifest file, replace it.
+        if 'rttm_filename' in item:
+            item['rttm_file'] = item.pop('rttm_filename')
+        elif 'rttm_filepath' in item:
+            item['rttm_file'] = item.pop('rttm_filepath')
         else:
-            item['audio_file'] = os.path.expanduser(item['audio_file'])
+            raise ValueError(
```

Comment:
can we use a default 'None' value for `item['rttm_file']` so that users don't have to manually set it to None during inference?

Reply:
The way we assign None, now

`item['rttm_file'] = None` is copied from https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/parts/preprocessing/manifest.py#L165